### PR TITLE
Update readme and define subprojects

### PR DESCRIPTION
**Pull request by qiujian16**
Signed-off-by: Jian Qiu <jqiu@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,6 +34,21 @@ The following repositories describe the underlying API and controllers for the a
 * https://github.com/open-cluster-management-io/multicloud-operators-subscription
 * https://github.com/open-cluster-management-io/multicloud-operators-channel
 
+### GRC: Governance, Risk and Compliance across Kubernetes clusters
+
+* Use prebuilt security and configuration controllers to enforce policy on Kubernetes configuration, identity and access management (IAM), Center for Internet Security (CIS), and certificate management across your clusters.
```
Comment (mikeshng) [inline README.md:39]: @dhaiducek could you please take a look at this change around GRC. I am not sure its accurate to mention things like IAM here. Thanks.

File: README.md [hunk 2]
```diff
@@ -9,17 +9,20 @@ The open-cluster-management architecture uses a hub - agent model. The hub centr
...
 
 * https://github.com/open-cluster-management-io/api
 * https://github.com/open-cluster-management-io/registration
 * https://github.com/open-cluster-management-io/work
+* https://github.com/open-cluster-management-io/placement
+* https://github.com/open-cluster-management-io/registration-operator
+* https://github.com/open-cluster-management-io/addon-framework
 
-### Delivery, upgrade, and configuration of applications on Kubernetes clusters
+### Application: Delivery, upgrade, and configuration of applications on Kubernetes clusters
```
Comment (mikeshng) [inline README.md:25]: I suggest we use `Application Lifecycle` similar to `Cluster Lifecycle`

File: README.md [hunk 3]
```diff
@@ -31,6 +34,21 @@ The following repositories describe the underlying API and controllers for the a
 * https://github.com/open-cluster-management-io/multicloud-operators-subscription
 * https://github.com/open-cluster-management-io/multicloud-operators-channel
 
+### GRC: Governance, Risk and Compliance across Kubernetes clusters
```
Comment (dhaiducek) [inline README.md:37]: Aligning with downstream, this should be "Governance:" instead of "GRC:"

File: README.md (hunk 1 above)
Comment (dhaiducek) [reply README.md:39]: Correct. For upstream this should probably only say "to enforce policies on Kubernetes configuration across your clusters."
The IAM and Certificate controllers were not brought over, and CIS is delivered through the compliance operator (not out of the box).

**Pull request closed without merging**